app/models/form/lettings/questions/location_id.rb

```diff
@@ -31,6 +31,11 @@ def displayed_answer_options(lettings_log, _user = nil)
 
     scheme_location_ids = lettings_log.scheme.locations.visible.confirmed.pluck(:id)
     answer_options.select { |k, _v| scheme_location_ids.include?(k.to_i) }
+                  .sort_by { |_, v|
+                    name = v["hint"].match(/[a-zA-Z].*/).to_s
+                    number = v["hint"].match(/\d+/).to_s.to_i
+                    [name, number]
+                  }.to_h
```

Comment:
Can you add some tests for this ordering?

Reply:
Done